package/crossplane.yaml.template

```
@@ -3,11 +3,18 @@ kind: Provider
 metadata:
   name: provider-cloudscale
   annotations:
-    meta.crossplane.io/maintainer: VSHN <info@vshn.net>
+    meta.crossplane.io/maintainer: VSHN <info@vshn.ch>
     meta.crossplane.io/source: github.com/vshn/provider-cloudscale
     meta.crossplane.io/license: Apache-2
     meta.crossplane.io/description: |
-      Crossplane provider for managing resources on cloudscale.ch
+      VSHN's Crossplane provider to manage Cloudscale services in Kubernetes.
+    meta.crossplane.io/readme: |
+      `provider-cloudscale` is the Crossplane infrastructure provider for
+      [Cloudscale](https://www.cloudscale.ch/), provided by [VSHN](https://www.vshn.ch/).
```

```suggestion
      [cloudscale.ch](https://www.cloudscale.ch/), provided by [VSHN](https://www.vshn.ch/).
```

They are called like that